webapps/package-lock.json

```
@@ -2394,7 +2394,7 @@
       }
     },
     "camunda-webapp-translations": {
-      "version": "https://github.com/camunda/camunda-webapp-translations.git#352c0d0984e70aa0fdc91d85e1352b9b1960e852",
+      "version": "https://github.com/camunda/camunda-webapp-translations.git#db7bb3e92c5f125dc732f52bc19d6854a6c1f73f",
```

```suggestion
      "version": "https://github.com/camunda/camunda-webapp-translations.git#126c493650f4d50f4faa7c19c78f64ef30be5d46",
```
Shouldn't it be this one?